Library/Homebrew/cmd/install.rb

```diff
@@ -267,6 +267,7 @@ def install
     # Need to rescue before `FormulaUnavailableError` (superclass of this)
     # is handled, as searching for a formula doesn't make sense here (the
     # formula was found, but there's a problem with its implementation).
+    puts e.backtrace
```

Comment:
I'm going to be picky and suggest:

```suggestion
    $stderr.puts e.backtrace
```